workbench-core/projects-ui/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@aws/workbench-core-projects-ui",
```

Comment:
Thoughts on naming this `workbench-core-accounts-ui`? 

I'm thinking we can put the following API components into one package.

`accounts, cost indexes, projects`

Then we can do the same thing with UI components.

If we don't group them  together, we'll end up with a package for each resource entity, which is probably too many packages.